raiden-dapp/src/views/NotificationPanel.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div id="notification-panel-wrapper">
```

Comment:
could we make this `class="notification-panel"` and the other class maybe `class="notification-panel__content"` ?

Reply:
Yes sir!